### PR TITLE
Transport delta ticks via delta key.

### PR DESCRIPTION
**Pull request by cramforce**
Removing the old and weird way to do it.

First item of  #8216

### CODE REVIEW

File: src/service/performance-impl.js [hunk 1]
```diff
@@ -232,21 +232,26 @@ export class Performance {
...
    *    relative start for this tick.
    * @param {number=} opt_value The time to record the tick at. Optional, if
-   *    not provided, use the current time. You probably want to use
-   *    `tickDelta` instead.
+   *    not provided, use the current time.
+   * @param {number=} tickDelta The delta. Call tickDelta instead of setting
+   *     this directly.
    */
-  tick(label, opt_from, opt_value) {
+  tick(label, opt_from, opt_value, opt_delta) {
```
Comment (lannka) [inline src/service/performance-impl.js]: let's remove `opt_from` and `opt_value` as well?

Comment (cramforce) [inline src/service/performance-impl.js]: In this case we can, because no other binaries use it. Otherwise we'd need to be careful.

Comment (cramforce) [inline src/service/performance-impl.js]: Done

**Comment by cramforce** [comment]
Please re-approve

**Pull request merged**